style.css

```diff
@@ -493,3 +494,231 @@ form ul li button:hover {
   background-color: #fff5e1;
   border: 1px solid #f55800;
 }
+
+/* -----------------------------css for desktop or bigger screens-------- */
+@media only screen and (min-width: 768px) {
+  html {
+    width: 100%;
+  }
+
+  .menu {
+    display: none;
+  }
+
```

Comment:
- [ ] Currently, there is a horizontal overflow on the page. Kindly fix it for a better user experience. 💯️

<img src="https://user-images.githubusercontent.com/17243325/214769898-d5d688a1-046d-4f01-8342-7214880b0b0a.png" width="500">